crates/bevy_core_pipeline/src/core_3d/main_transparent_pass_3d_node.rs

```diff
@@ -81,11 +68,9 @@ impl ViewNode for MainTransparentPass3dNode {
                 label: Some("reset_viewport_pass_3d"),
                 color_attachments: &[Some(target.get_color_attachment(Operations {
                     load: LoadOp::Load,
-                    store: StoreOp::Store,
+                    store: true,
```

Comment:
Why is this a bool value and not an enum?

Reply:
Fixed, didn't notice it because it was behind a cfg :P